apps/mobile/src/store/keychains/bitcoin/bitcoin-keychains.read.ts

```diff
@@ -132,3 +132,11 @@ export function useBitcoinAddresses() {
     [accounts]
   );
 }
+
+export function deriveBitcoinAccountFromAddress(
```

Comment:
can we rename this function because there's no derivation. `findAccountFromAddress`? `drive` implies there is some derivation going on, and there isn't.